sources/rest_api/config_setup.py

```diff
@@ -158,24 +158,36 @@ def setup_incremental_object(
     Optional[Incremental[Any]], Optional[IncrementalParam], Optional[Callable[..., Any]]
 ]:
     incremental_params: List[str] = []
-    for key, value in request_params.items():
-        if isinstance(value, dict) and value.get("type") == "incremental":
-            incremental_params.append(key)
+    for param_name, param_config in request_params.items():
+        if (
+            isinstance(param_config, dict)
+            and param_config.get("type") == "incremental"
+            or isinstance(param_config, dlt.sources.incremental)
```

Comment:
previously, users could pass an unlimited number of `dlt.sources.incremental` objects, the first one was taken and the others were ignored.